schema/bom-1.6.schema.json

```diff
@@ -913,6 +919,12 @@
           "description": "Specifies the scope of the component. If scope is not specified, 'required' scope SHOULD be assumed by the consumer of the BOM.",
           "default": "required"
         },
+        "isExtraneous": {
```

Comment:
If the environment provides a component, then it is best to capture it under formulation instead of having such boolean flags.

Reply:
why should it be under "formulation"? 
form,ulation is build-time, while this domain here may also be runtime.

Reply:
Wouldn't such runtime-only components not captured in an OBOM?

Reply:
how would you make it clear in an obom, that the build result descibed in an SBOM needs windows 11 patch 23H2 on runtime? Or that it  requires python 3.11 or later as an interpreter? 
I dont know ... this is why this PR exists, to discuss these options and maube provide a solution :-) 

read more here in the sectiopn "usecases" of https://github.com/CycloneDX/specification/issues/321

Reply:
Thank you. I should've read the use cases first, which helped. I would have used two separate documents and linked them via external references. However, I see the appeal of having a single combined document for both SBOM and its operating environment. Perhaps component.type = "platform" indicates runtime without the need for the additional boolean flag?